ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardResources.resource

```diff
@@ -18,9 +18,9 @@ Open QuickStart Element in Resource Section By Name
     [Documentation]     It opens the element by enabling Quick Start filter in Resources Section of ODS Dashboard.
     [Arguments]     ${element}
     Select Checkbox     //input[@id="quickstart--check-box"]
-    Wait Until Page Contains Element    ${RES_CARDS_XP}\[@id="${element}"]
-    Click Link          ${RES_CARDS_XP}\[@id="${element}"]//a
-    Wait Until Page Contains ELement    //div[@class="pf-c-drawer__panel-main"]     5
+    Wait Until Page Contains Element    xpath=//div[@id="${element}"]//a[text()="Open"]
```

Comment:
why don't you update the value of `${RES_CARDS_XP}` instead of changing `//div` everywhere?

Reply:
Because "${RES_CARDS_XP}" is used in other places and files (it's not even defined in this file), and also I think that "div" is easier to read and understand than "${RES_CARDS_XP}"

Reply:
>Because "${RES_CARDS_XP}" is used in other places and files (it's not even defined in this file)


probably it needs to be changed also in other places.


> and also I think that "div" is easier to read and understand than "${RES_CARDS_XP}"


for maintanance purposes, it's easier to change the value of one variable rather than replacing all the instances of the outdated path, no?

Reply:
Done!